unblob/processing.py

```diff
@@ -143,6 +143,17 @@ def __init__(
         self.size = size
         self.result = result
 
+        self.extract_dir = self._get_extract_dir()
+
+        assert self.extract_dir.is_absolute()
+
+    def _get_extract_dir(self) -> Path:
+        """Extraction dir under root with the name of path."""
+        relative_path = self.task.path.relative_to(self.task.root)
+        extract_name = relative_path.name + self.config.extract_suffix
```

Comment:
just wondering, but do not we want to distinguish between _carve and _extract directory?

Reply:
I would prefer them to be different, but would make the change in another PR.

Reply:
#326